content/stylesheets/components/_banner.scss

```diff
@@ -0,0 +1,6 @@
+.app-banner {
+  margin: 10px 0;
+  padding: 10px 15px;
+  background-color: color-mix(in srgb, currentColor 5%, transparent);
```

Comment:
I wonder if this should be a bit stronger for drawing attention, its not as attention drawing now especially in light mode:

<img width="536" alt="image" src="https://github.com/user-attachments/assets/ce6dc6bd-7c56-4f28-88ef-022036fe2931">

Previously:

<img width="1215" alt="image" src="https://github.com/user-attachments/assets/0fc173f9-8d4a-44b4-8ccd-ab5a96090cf6">

thoughts?

Reply:
Sounds reasonable to me - 

<img width="504" alt="image" src="https://github.com/user-attachments/assets/43a1c357-3c32-4e41-9187-8a262f65b8d0">

Is this enough?

Reply:
![image](https://github.com/user-attachments/assets/61a023f4-5afa-47ed-a12e-837c77ede65f)